manage_sections/views.py

```diff
@@ -142,7 +142,15 @@ def create_section_form(request):
         logger.debug('Total time for get_course is ={} for course {}'.format(time.time() - start, canvas_course_id))
         total_students_size = 0
         if course:
-            total_students_size = course['total_students']
+            if 'total_students' not in course:
+                logger.debug('total_students not in cached object, flushing and refetching object..')
+                # this is probably  a cached object without the count. Clear the cache object and re-fetch the course
+                canvas_api_helper_courses.delete_cache(canvas_course_id=canvas_course_id)
+                course = canvas_api_helper_courses.get_course(canvas_course_id, **kwargs)
+
+            if course and 'total_students' in course:
```

Comment:
There's already an `if course` at line 144, so this line could just be `if 'total_students' in course:'.

Reply:
Yeah. the reason I am checking for both is bcoz we make a fresh call for fetching  course in 149. The chances of it being Null is pretty slim though..

Reply:
Ah, right! Never mind! :-)